packages/wallets/src/features/cfd/screens/Jurisdiction/JurisdictionCard/JurisdictionCard.tsx

```diff
@@ -71,10 +71,14 @@ const JurisdictionCard: React.FC<TJurisdictionCardProps> = ({ isSelected, jurisd
                 onSelect(jurisdiction);
             }}
         >
-            {!isFlipped && isOverHeaderAvailable && <JurisdictionCardTag tag={overHeader || ''} />}
             <React.Fragment>
                 <div className='wallets-jurisdiction-card-front'>
-                    <div className='wallets-jurisdiction-card-front__label'>{header}</div>
+                    {isOverHeaderAvailable && <JurisdictionCardTag tag={overHeader || ''} />}
+                    <div className='wallets-jurisdiction-card-front__label'>
+                        <WalletText align='center' size='lg' weight='bold'>
+                            {header}
+                        </WalletText>
+                    </div>
```

Comment:
This fixes issue where the top tag flickers when the card is flipped:
## Before:
https://github.com/binary-com/deriv-app/assets/103016120/c7813f27-4bde-4be8-94a4-d52ecdb43297

## After:
https://github.com/binary-com/deriv-app/assets/103016120/f5f17482-b834-487b-bf61-5423224675db